.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,36 @@
+<!--- Provide a general summary of your changes in the Title above -->
+
+## Description
+<!--- Describe your changes in detail -->
+
+## Motivation and Context
+<!--- Why is this change required? What problem does it solve? -->
+<!--- If it fixes an open issue, please link to the issue here -->
+
+## Testing
+<!--- Please describe in detail how you tested your changes -->
+<!--- Include details of your testing environment, and the tests you ran to -->
+<!--- see how your change affects other areas of the code, etc. -->
+
+## Screenshots (if appropriate)
+
+## Types of changes
+<!--- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
+- [ ] Bug fix (non-breaking change which fixes an issue)
+- [ ] New feature (non-breaking change which adds functionality)
+
+## Checklist
+<!--- Go over all the following points, and put an `x` in all the boxes that apply -->
+<!--- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
+- [ ] My code follows the code style of this project
+- [ ] My change requires a change to the documentation
+- [ ] I have updated the documentation accordingly
```

Comment:
Does this mean javadoc or developer guide?

Reply:
Should always mean javadoc.

Not many (if any) changes post GA should require updates to the dev guide. Changes that do will likely be taken on by our team separately.

Reply:
I wouldn't expect a customer to update the developer guide too so +1 for clarifying this as javadoc.

Reply:
:+1: